### PR TITLE
JS Fundamentals - Part 1: Add link to explain difference between var, let and const | Add consistent naming to operators

### PR DESCRIPTION
**Pull request by SauravJalui**
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [ ] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [ ] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

**2. This PR:**

**3. Additional Information:**

### CODE REVIEW

File: foundations/javascript_basics/fundamentals-1.md [hunk 1]
```diff
@@ -12,12 +12,12 @@ This section contains a general overview of topics that you will learn in this l
...
-* What is the difference between `==` and `===`?
+* What is the difference between loose equality(==) and strict equality(===)?
 * What are operator precedence values?
-* What are the increment/decrement operators?
-* What is the difference between prefixing and post-fixing them?
-* What are assignment operators?
-* What is the "Unary +" Operator?
+* What are the increment(++)/decrement(--) operators?
+* What is the difference between prefixing and postfixing them?
+* What are assignment(=) operators?
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:19]: I think a part of being able to answer this question confidently is knowing that the assignment operator is `=`. I don't like giving it away in the question.

File: foundations/javascript_basics/fundamentals-1.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ You can think of variables as simply "storage containers" for data in your code.
...
+The above tutorial mentioned this, but it's important enough to note again: `let` and `const` are both relatively new ways to declare variables in JavaScript. <span id="avoid-var">In _many_ tutorials (and code) across the internet you're likely to encounter `var` statements. Don't let it bother you! There's nothing inherently wrong with `var`, and in most cases `var` and `let` behave the same way. But sometimes the behavior of `var` is _not_ what you would expect. Just stick to `let` (and `const`) for now.</span> The precise differences between `var` and `let` is explained [here](https://javascript.info/var).
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:62]: Generally, we like to avoid using "here" when we link to external resources. Could you rephrase this so that it sort of "gives credit" to the site in the text? Furthermore, this might be a great first link for the additional resources section as opposed to being shoved inline here. As a learner, I'd wonder whether it was required for me to read or not (it's not).

File: foundations/javascript_basics/fundamentals-1.md [hunk 3]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
-* <a class="knowledge-check-link" href="https://javascript.info/operators#remainder">How does the `%` operator work?</a>
-* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">Explain the difference between `==` and `===`.</a>
-* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">When would you receive a `NaN` result?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">How do you increment and decrement a number?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">Explain the difference between prefixing and post-fixing increment/decrement operators.</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#string-concatenation-with-binary">What happens when you add numbers and strings together?</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#remainder">How does the Modulus(%)* operator work?</a>
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:110]: Can we change to Modulo?

Also, instead of putting an asterisk, just put the "remainder" hint inline.

`How does the Modulo (%), or Remainder, operator work?`

File: foundations/javascript_basics/fundamentals-1.md [hunk 4]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
+* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">Explain the difference between loose equality(==) and strict equality(===).</a>
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:111]:
```suggestion
* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">Explain the difference between loose equality (==) and strict equality (===).</a>
```

File: foundations/javascript_basics/fundamentals-1.md [hunk 5]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
+* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">When would you receive a `NaN`(Not a Number) result?</a>
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:112]:
```suggestion
* <a class="knowledge-check-link" href="https://www.w3schools.com/js/js_numbers.asp">When would you receive a `NaN` (Not a Number) result?</a>
```

File: foundations/javascript_basics/fundamentals-1.md [hunk 6]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
+* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">How do you increment(++) and decrement(--) a number?</a>
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:113]: The hints here are, again, part of being able to answer this question confidently. It gives too much away.

File: foundations/javascript_basics/fundamentals-1.md [hunk 7]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
+* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">Explain the difference between prefixing and postfixing increment(++)/decrement(--) operators.</a>
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:114]: Same as above, but the spell correction on "postfixing" should stay.

File: foundations/javascript_basics/fundamentals-1.md [hunk 8]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
 * <a class="knowledge-check-link" href="https://javascript.info/operators#operator-precedence">What is operator precedence and how is it handled in JS?</a>
 * <a class="knowledge-check-link" href="#access-devTools-console">How do you access developer tools and the console?</a>
 * <a class="knowledge-check-link" href="#console-log">How do you log information to the console?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#numeric-conversion-unary">What does unary plus operator do to string representations of integers?</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#numeric-conversion-unary">What does unary plus(+) operator do to string representations of integers?</a>
+
+\* `%` is also called the remainder operator
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:120]: Remove in favor of the above

File: foundations/javascript_basics/fundamentals-1.md [hunk 9]
```diff
@@ -106,13 +106,15 @@ This section contains questions for you to check your understanding of this less
...
+* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">How do you increment(++) and decrement(--) a number?</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#increment-decrement">Explain the difference between prefixing and postfixing increment(++)/decrement(--) operators.</a>
 * <a class="knowledge-check-link" href="https://javascript.info/operators#operator-precedence">What is operator precedence and how is it handled in JS?</a>
 * <a class="knowledge-check-link" href="#access-devTools-console">How do you access developer tools and the console?</a>
 * <a class="knowledge-check-link" href="#console-log">How do you log information to the console?</a>
-* <a class="knowledge-check-link" href="https://javascript.info/operators#numeric-conversion-unary">What does unary plus operator do to string representations of integers?</a>
+* <a class="knowledge-check-link" href="https://javascript.info/operators#numeric-conversion-unary">What does unary plus(+) operator do to string representations of integers?</a>
```
Comment (fortypercenttitanium) [inline foundations/javascript_basics/fundamentals-1.md:118]: Instead of just putting a plus, can we give an example?
`What does unary plus operator do to string representations of integers? Ex: +"12"`

**Review by fortypercenttitanium — CHANGES_REQUESTED** [review]
Thanks for your patience on this one. I have a few changes I'm suggesting. Thank you for your contribution!

File: foundations/javascript_basics/fundamentals-1.md (hunk 1 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:19]: Makes sense. I wanted the questions to be consistent throughout, hence the brackets with the operators in them. I agree with the point you're making. I'll update the assignment operator question.

File: foundations/javascript_basics/fundamentals-1.md (hunk 2 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:62]: I've rephrased the sentence and moved it to the additional resources section.

I agree it might not be as important as a beginner to learn the difference between let and const. However, if someone wants to, they can do so from the additional resources section.

File: foundations/javascript_basics/fundamentals-1.md (hunk 3 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:110]: Sure, I've changed the sentence and added the hint inline.

File: foundations/javascript_basics/fundamentals-1.md (hunk 4 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:111]: fixed.

File: foundations/javascript_basics/fundamentals-1.md (hunk 5 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:112]: fixed.

File: foundations/javascript_basics/fundamentals-1.md (hunk 8 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:120]: done.

File: foundations/javascript_basics/fundamentals-1.md (hunk 7 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:114]: done.

File: foundations/javascript_basics/fundamentals-1.md (hunk 6 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:113]: removed all hints.

File: foundations/javascript_basics/fundamentals-1.md (hunk 9 above)
Comment (SauravJalui) [reply foundations/javascript_basics/fundamentals-1.md:118]: fixed and added an example.

**Review by fortypercenttitanium — APPROVED** [review]
Great work, thank you!
